salt/pillar/reviewer-suggestions.sls

```diff
@@ -14,3 +14,17 @@ reviewer_suggestions:
         password: barpass
         host: 127.0.0.1
         port: 5432
+
+    auth:
+        auth0_domain: elife.auth0.com
+        auth0_client_id: s0mwbpPUuDBuYAAFMxYUT6J9xF6pn67O
+        valid_domains: elifesciences.org
```

Comment:
The `auth0_domain` is the API entry point for Auth0.

The `valid_domains` is actually something I added to only allow access with emails from a specific domain or a fixed list of emails. It is possible to do that via auth0 rules / hooks but it was simpler without auth0 being involved in that. Perhaps `valid_email_domains` would be a better name.